src/Playroom/SettingsPanel/SettingsPanel.tsx

```diff
@@ -24,21 +24,18 @@ const getKeyBindings = () => {
   const metaKeySymbol = isMac() ? '⌘' : 'Ctrl';
   const altKeySymbol = isMac() ? '⌥' : 'Alt';
 
-  // Sort shortcuts alphabetically
-  /* eslint sort-keys: "error" */
   return {
-    'Add cursor to next line': [metaKeySymbol, altKeySymbol, '↓'],
-    'Add cursor to prev line': [metaKeySymbol, altKeySymbol, '↑'],
-    'Duplicate line down': ['⇧', altKeySymbol, '↓'],
-    'Duplicate line up': ['⇧', altKeySymbol, '↑'],
     'Format code': [metaKeySymbol, 'S'],
```

Comment:
Thoughts on shifting format below "Wrap selection in tag"?

Reply:
Yeah, good idea I reckon.